hugo/layouts/robots.txt

```diff
@@ -1,4 +1,6 @@
 User-agent: *
 Allow: /
 
```

Comment:
The well-hidden but mostly-canonical reference site for `robots.txt` mentions blank lines being a problem:

> you may not have blank lines in a record, as they are used to delimit multiple records.

-- https://www.robotstxt.org/robotstxt.html

Reply:
Fixed!